WEB-408: Improve Configuration Wizard popup UI

AnvayKharb opened the pull request:
_This PR improves the UI of the Configuration Wizard popup to make it cleaner, better aligned, and visually consistent with the rest of the application._

### Changes Made

**Aligned all buttons properly using a consistent layout.**

Added proper padding and spacing inside the popup.

Adjusted spacing between title, description, progress bar, and buttons.

Resized or restyled the “Close Config. Wizard” button to match other buttons and reduce visual imbalance.

Kept the UI simple — no extra fancy components, just better structure and clarity.

### Before 

<img width="1304" height="828" alt="Screenshot 2025-11-05 at 10 35 36 PM" src="https://github.com/user-attachments/assets/1b0e46d3-a218-45d8-972d-e66b06b01932" />


### After 
<img width="943" height="515" alt="Screenshot 2025-11-10 at 11 07 45 PM" src="https://github.com/user-attachments/assets/347ccce4-afd9-4d26-ae97-2b7e1144d543" />


### How to Test

Go to Home → Configuration Wizard popup.

Check alignment of buttons and spacing between elements.

Verify that the popup still works functionally with no layout or console errors.

Confirm responsiveness if applicable.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Style**
  * Redesigned the configuration wizard with a clearer containerized layout and consolidated action area.
  * Reflowed content into a descriptive content section and a labeled progress area with updated progress styling.
  * Introduced a responsive buttons grid for consistent button sizing, spacing, and wrapping across mobile, tablet, and desktop.
  * Preserved existing translations and button semantics.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

AnvayKharb commented:
@gkbishnoi07 please take a look here

steinwinde reviewed (dismissed):
Please make the wizard styles more inline with the rest of the site. E.g. why should the font-weight be different from the way we usually render button labels? E.g. an almost transparent button with a red font ("warn") does (for my opinion) not look professional - I'm not aware we have anything like this anywhere in the app; the same button visible on all subsequent wizard modal windows looks different.

AnvayKharb committed:
WEB-408: Improve Configuration Wizard UI

- Redesigned dialog layout with centered content and improved spacing
- Changed button grid to display 3 columns (33.333% each) with responsive breakpoints
- Added progress bar section with cleaner styling
- Improved button styles to match site standards (removed custom font-weight, changed Close button from stroked-warn to standard raised button)
- Enhanced visual hierarchy and professional appearance
- Responsive design for tablets and mobile devices

AnvayKharb commented:
hey @steinwinde I've just updated my pr as per your suggestion.

AnvayKharb commented:
@steinwinde you can check now

IOhacker commented:
@AnvayKharb could you please update the screenshots in the PR description

AnvayKharb commented:
@IOhacker heyy I've just updated my pr

gkbishnoi07 reviewed (approved):
LGTM

AnvayKharb commented:
@steinwinde please consider

AnvayKharb commented:
@gkbishnoi07 please merge this if no further changes are required.
